hack/helm-reference-gen/main.go

```diff
@@ -159,11 +159,14 @@ func GenerateDocs(yamlStr string) (string, error) {
 		return "", err
 	}
 
-	enterpriseSubst := strings.ReplaceAll(strings.Join(children, "\n\n"), "[Enterprise Only]", "<EnterpriseAlert inline />")
+	docsStr := strings.Join(children, "\n\n")
+	docsStr = strings.ReplaceAll(docsStr, "[Enterprise Only]", "<EnterpriseAlert inline />")
+	// Remove https://developer.hashicorp.com prefix from links because docs linting requires it.
+	docsStr = strings.ReplaceAll(docsStr, "https://developer.hashicorp.com/", "/")
```

Comment:
this should work. just wanted to call out that consul's docs live at `https://developer.hashicorp.com/consul/docs`, does this account for the `/consul`?

Reply:
just confirmed that it does, feel free to merge when ready!